src/similarity.ts

```diff
@@ -47,3 +50,23 @@ export function computeSimilarity(
   // If we couldn't compare JSON, then we'll assume they don't match.
   return +Infinity;
 }
+
+/**
+ * Strips out headers that are likely to result in false negatives.
+ */
+function stripExtraneousHeaders(headers: Headers): Headers {
+  const safeHeaders: Headers = {};
+  for (const key of Object.keys(headers)) {
```

Comment:
Just a suggestion:
```
const blackList = ["accept", "user-agent", "host", "connecton"];  // could be declared at the top?
const safeHeaders = Object.keys(headers).filter(header => !blackList.includes(header));
return safeHeaders.reduce((acc, header) => ({ ...acc, [header]: headers[header]}), {});
```

Reply:
I find this a lot harder to read (especially taking the perspective of someone who doesn't write JS every day).

I think putting the blacklist into a global constant might make sense—although maybe it'd better to wait until it's configurable?

I'll leave it as is for now in the interest of time & simplicity. Thanks for the suggestion though!